source/margarita/minions.rst

```diff
@@ -0,0 +1,42 @@
+.. _minions:
+
+Minions
+=======
+
+Within an environment, there's one server, the *master*, that is
+running ``salt-master``.
+
+All the servers in the environment also run ``salt-minion`` and are
+referred to collectively as *minions*.
+
+What each minion does is defined by adding to *roles* in the
```

Comment:
I'd maybe say, "What each minion does is defined by editing the list of *roles*...", thought it might be good to add a caveat that removing a role from a minion doesn't remove the services that have already been deployed.